packages/python-runner/src/PythonFileEditor.tsx

```diff
@@ -131,7 +131,8 @@ export class PythonFileEditor extends DocumentWidget<
    */
   private createOutputAreaWidget = (): void => {
     // Add dockpanel wrapper for output area
-    this.dockPanel = new DockPanelSvg();
+    const dockPanelOptions: DockPanel.IOptions = { tabsMovable: false };
+    this.dockPanel = new DockPanelSvg(dockPanelOptions);
```

Comment:
is there a reason we can't just do this "inline" like below and avoid an extra import?
```suggestion
    this.dockPanel = new DockPanelSvg({ tabsMovable: false });
```

Reply:
I guess typescript got me used to define a type for everything 😄